tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,677 @@
+"""
+.. _tut-ssvep:
+
+==========================================================
+Frequency-tagging: Basic analysis of an SSVEP/vSSR dataset
+==========================================================
+
+In this tutorial we compute the frequency spectrum and quantify signal-to-noise
+ratio (SNR) at a target frequency in EEG data recorded during fast periodic
+visual stimulation (FPVS) at 12 Hz and 15 Hz in different trials.
+Extracting SNR at stimulation frequency is a simple way to quantify frequency
+tagged responses in MEEG (a.k.a. steady state visually evoked potentials,
+SSVEP, or visual steady-state responses, vSSR in the visual domain,
+or auditory steady-state responses, ASSR in the auditory domain).
+
+For a general introduction to the method see
+`Norcia et al. (2015) <https://doi.org/10.1167/15.6.4>`_ for the visual domain,
+and `Picton et al. (2003) <https://doi.org/10.3109/14992020309101316>`_ for
+the auditory domain.
+
+**Data and outline:**
+
+We use a simple example dataset with frequency tagged visual stimulation:
+N=2 participants observed checkerboard patterns inverting with a constant
+frequency of either 12.0 Hz of 15.0 Hz.
+32 channels wet EEG was recorded.
+(see :ref:`ssvep-dataset` for more information).
+
+We will visualize both the power-spectral density (PSD) and the SNR
+spectrum of the epoched data,
+extract SNR at stimulation frequency,
+plot the topography of the response,
+and statistically separate 12 Hz and 15 Hz responses in the different trials.
+Since the evoked response is mainly generated in early visual areas of the
+brain the statistical analysis will be carried out on an occipital
+ROI.
+
+.. contents:: local
+   :depth: 2
+"""  # noqa: E501
+# Authors: Dominik Welke <dominik.welke@web.de>
+#          Evgenii Kalenkovich <e.kalenkovich@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+import numpy as np
+from scipy.stats import ttest_rel
+
+###############################################################################
+# Data preprocessing
+# ------------------
+# Due to a generally high SNR in SSVEP/vSSR, typical preprocessing steps
+# are considered optional. This doesn't mean, that a proper cleaning would not
+# increase your signal quality!
+#
+# * Raw data have FCz reference, so we will apply common-average rereferencing.
+#
+# * We  will apply a 0.1 highpass filter.
+#
+# * Lastly, we will cut the data in 20 s epochs corresponding to the trials.
+#
+#
+
+# Load raw data
+data_path = mne.datasets.ssvep.data_path()
+bids_fname = data_path + '/sub-02/ses-01/eeg/sub-02_ses-01_task-ssvep_eeg.vhdr'
+
+raw = mne.io.read_raw_brainvision(bids_fname, preload=True, verbose=False)
+raw.info['line_freq'] = 50.
+
+# Set montage
+montage = mne.channels.make_standard_montage(
+    'easycap-M1',
+    head_size=0.095)  # head_size parameter default = 0.095
```

Comment:
If this is just the default value can it be removed?
```suggestion
montage = mne.channels.make_standard_montage( 'easycap-M1)
```